### ISSUE TITLE
[Bug Report] Certain Fonts not showing even when installed

### ISSUE THREAD

**Issue by Bagel94** [ChurchApps/FreeShow#768]
**Describe the bug**
I have been using some fonts that I got from the Google Fonts Library and every now and then when there is an update, FreeShow will not see the fonts even though they are installed, and other programs have no issue finding them. Only way to get FreeShow to see the fonts again, is to uninstall the font and then install it again. After do this, restart FreeShow, it then sees the re-installed fonts.

One particular font is "BEBAS NUE REGULAR"

**Reproducing**
Happens at random, last 2-3 updates have not caused it to happen until now. But seems to have started once the version # hit the 1.1.8 series and above.

**Version (Optional)**
 - OS: Windows 10/11
 - FreeShow Version: 1.2.5

**Comment by Bagel94** [comment]
Correction, after reinstalling the font and restarting FreeShow, the font did show up. Now having shut everything down, fonts are missing again.

FreeShow even says that's the font selected, even though it is not showing up in the drop-down list. Nor is text box font matching.

![image](https://github.com/user-attachments/assets/bf738c75-1a0c-4dbe-8577-89445a1fd534)

**Comment by vassbo** [comment]
Are you sure this did not happen before version 1.1.8?
It could be an issue with the font-package: https://www.npmjs.com/package/font-list
I made a small change now, we'll see if that changes it.

**Comment by Bagel94** [comment]
Hmm, I don't recall this happening in the versions below that.

**Comment by vassbo** [comment]
Check if it's better here: https://github.com/ChurchApps/FreeShow/releases/tag/v1.2.6-beta.1

**Comment by Bagel94** [comment]
Installed this version. After opening FreeShow, the program did not initially detect the font. So, I did the fix I had been using, reinstall the font. After reopening FreeShow, the font did show up and persisted through a couple closers of the program.

If it seems to disappear again, I'll report back here about it.

**Comment by vassbo** [comment]
All right, thanks. It seems to be an issue that I can't do much about unfortunately. Might be releated to the font-list package if it persists.

**Comment by Bagel94** [comment]
> All right, thanks. It seems to be an issue that I can't do much about unfortunately. Might be related to the font-list package if it persists.

So, an update... It seems like these issues only applies to Windows 11.

Here are the OS's I use FreeShow on:
Windows 11 - My desktop
Windows 10 - My laptop
Linux Mint - Church PC

Out of all 3 of these computers. My windows 11 machine is the only one that keeps losing fonts inside FreeShow. My laptop nor the churches PC have this issue. Tried out the Church PC earlier this morning, and just tested the new Beta on my laptop a moment ago, and both still have the font.

**Comment by Bagel94** [comment]
I did some searching and found a forum post about someone having issues with whatever program they had, and that it was losing fonts on windows 11. Someone commented on it and asked if the "OP" had installed the font's as "current user" or as "all users".

So, I have attempted to install the fonts as "all users", and I'll report back if FreeShow holds on to the fonts this time or if it continues to lose them. Currently I have tried just a restart of FreeShow and a restart of my PC and at this time, the fonts are still there and are usable.
